### PR TITLE
consistencyGroupPrefix is set as hidden in OCP UI

### PR DESCRIPTION
**Pull request by nitishkumar4**
## Pull request type

 

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature Enhancement
- [ ] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Community Operator listing 
- [ ] Other (please describe): 

## What is the current behavior?

- In OCP UI, Consistency Group Prefix is set as visible and available for user to modify.

## What is the new behavior?

- Consistency Group Prefix is set as hidden by default.
- Since consistency group prefix is fetched from the cluster, users don't need to explicitly provide a input for this unless they want to modify it. 

## How risky is this change?
- [x] Small, isolated change
- [ ] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

File: generated/installer/ibm-spectrum-scale-csi-operator-dev.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
       serviceAccountName: ibm-spectrum-scale-csi-operator
       containers:
       - name: operator
-        image: quay.io/ibm-spectrum-scale-dev/ibm-spectrum-scale-csi-operator:dev
```
Comment (Jainbrt) [inline generated/installer/ibm-spectrum-scale-csi-operator-dev.yaml:36]: we should not change the images in dev.yaml file to cp.icr.io path

**Review by Jainbrt — CHANGES_REQUESTED** [review]

**Review by Jainbrt — APPROVED** [review]

File: generated/installer/ibm-spectrum-scale-csi-operator-dev.yaml (hunk 1 above)
Comment (nitishkumar4) [reply generated/installer/ibm-spectrum-scale-csi-operator-dev.yaml]: reverted the changes in this file,

**Comment by vrushch** [comment]
consistencyGroupPrefix is now hidden in OLM Console. verified

**Review by madhuthorat — APPROVED** [review]
LGTM

**Pull request merged**